### ISSUE TITLE
Duplicate thumbnails with multiple cameras

### ISSUE THREAD

**Issue by jadedeane** [rasod/homebridge-kunalight#2]
Following up from #1, where the initial issue was my JSON formatting preventing accessory additions, there remains a problem with duplicate images.

It seems as though accessing http://homebridge:4000 or 4001 returns the same image.

<img width="1098" alt="Screen Shot 2019-11-06 at 3 45 43 PM" src="https://user-images.githubusercontent.com/11843794/68347662-8db44d00-00ac-11ea-8b2d-ac7fa22a5d1f.png">

![Screen Shot 2019-11-06 at 3 52 05 PM](https://user-images.githubusercontent.com/11843794/68347963-6d38c280-00ad-11ea-96ae-8347edf3e619.png)

```
		{
			"platform": "Camera-ffmpeg",
			"cameras": [
				{
					"name": "Garage East Camera",
					"videoConfig": {
						"source": "-i http://127.0.0.1:4000/thumbnail",
						"stillImageSource": "-i http://127.0.0.1:4000/thumbnail",
						"maxStreams": 1,
						"maxWidth": 1280,
						"maxHeight": 720,
						"maxFPS": 30
					}
				},
				{
					"name": "Garage West Camera",
					"videoConfig": {
						"source": "-i http://127.0.0.1:4001/thumbnail",
						"stillImageSource": "-i http://127.0.0.1:4001/thumbnail",
						"maxStreams": 1,
						"maxWidth": 1280,
						"maxHeight": 720,
						"maxFPS": 30
					}
				}
			]
		}
```

```
	"accessories": [
		{
			"accessory": "Kuna",
			"name": "Garage East Light",
			"email": "EMAIL",
			"password": "PASSWORD",
			"serial": "UNIQUE_SERIAL",
			"proxyThumbnail": true,
			"proxyPort": 4000,
			"polling": 1
		},
		{
			"accessory": "Kuna",
			"name": "Garage West Light",
			"email": "EMAIL",
			"password": "PASSWORD",
			"serial": "UNIQUE_SERIAL",
			"proxyThumbnail":true,
			"proxyPort": 4001,
			"polling": 1
		}
	]
```

**Comment by rasod** [comment]
hmm, I never tried with multiple cams. Let me try somethings, sorry for the delay.

**Comment by jadedeane** [comment]
> hmm, I never tried with multiple cams. Let me try somethings, sorry for the delay.

No worries, wasn't expecting "support" :) I don't write in any form of js, so hadn't really kicked the tires yet. Please let me know how I can help if needed.

**Comment by rasod** [comment]
@jadedeane I think i have fixed in the latest version 1.0. The configuration has changed slightly you need to use the serial number not keyword thumbnail when creating the camera. I don’t have multiple cameras myself so it is not fully tested. Fingers Crossed

**Comment by jadedeane** [comment]
> @jadedeane I think i have fixed in the latest version 1.0. The configuration has changed slightly you need to use the serial number not keyword thumbnail when creating the camera. I don’t have multiple cameras myself so it is not fully tested. Fingers Crossed

That worked perfectly. Really appreciate it!
